docs/modules/solr.md

```diff
@@ -0,0 +1,45 @@
+# Solr Container
+
+!!! note
+    This module is INCUBATING. While it is ready for use and operational in the current version of Testcontainers, it is possible that it may receive breaking changes in the future. See [our contributing guidelines](/contributing/#incubating-modules) for more information on our incubating modules policy.
+
+
+This module helps running [solr](https://lucene.apache.org/solr/) using Testcontainers.
+
+Note that it's based on the [official Docker image](https://hub.docker.com/_/solr/).
+
+## Usage example
+
+You can start a solr container instance from any Java application by using:
+
+```java
```

Comment:
It would be great if this example could be done via a 'codeinclude' of actual code from tests - this helps us keep documentation code up to date. You can see an example of this [here](https://github.com/testcontainers/testcontainers-java/blob/master/docs/modules/databases/mongodb.md#L10-L12).